crowbar_framework/config/initializers/crowbar_checks.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "crowbar/checks"
+
+unless ENV["SKIP_CHECKS"] && ENV["SKIP_CHECKS"] == "yes"
+  network_checks = Crowbar::NetworkChecks.new(Rails.logger)
+
+  unless network_checks.run_all_checks
+    raise "Crowbar network checks failed."
+  end
```

Comment:
The plan is to run the tests on each crowbar start, they take not much time but we make sure that we are in workable state. Actually I raise the error because we have no UI atm but when we have it the we just replace this raise with the line which triggers the UI page. When we throw now just a warning then I'm sure nobody cares about it.